config/crd/patches/webhook_in_drpolicies.yaml

```diff
@@ -12,5 +12,3 @@ spec:
           namespace: system
           name: webhook-service
           path: /convert
-      conversionReviewVersions:
```

Comment:
Do we even need this file now? Can this be removed?

Reply:
I think we need to keep it as it is part of the scaffolding.

Reply:
Is this and config/crd/patches/cainjection_in_drpolicies.yaml needed though, I am not seeing the scaffolding requirement here?

Reply:
so, this file was created when the ramen operator was scaffolded initially. I was just trying to revert the changes that we did when adding webhooks and this line was the only change. I tested now by deleting the file. It does not matter. we can safely delete the file I believe.